tests/drivers/build_all/sensor/i2c.dtsi

```diff
@@ -758,6 +758,11 @@ test_i2c_tcs3400: tcs3400@6b {
 	int-gpios = <&test_gpio 0 0>;
 };
 
+test_i2c_lsm9ds1: lsm9ds1@6b {
+	compatible = "st,lsm9ds1";
+	reg = <0x6b>;
+};
```

Comment:
This is reusing the same address as the node above it. Please move to the bottom and use a new address

Reply:
Done